content/en/docs/concepts/services-networking/service.md

```diff
@@ -14,24 +14,25 @@ weight: 10
 
 <!-- overview -->
 
-{{< glossary_definition term_id="service" length="short" >}}
+{{< glossary_definition term_id="service" length="short" prepend="In Kubernetes, a Service is" >}}
 
-With Kubernetes you don't need to modify your application to use an unfamiliar service discovery mechanism.
-Kubernetes gives Pods their own IP addresses and a single DNS name for a set of Pods,
-and can load-balance across them.
```

Comment:
any special reason to hide the IP and DNS details? too low-level?

Reply:
This conflates aspects of the Pod and Service APIs, which I think can be unhelpful.

Reply:
I think that this is explained later with more detail, so this SGTM